pkgs/by-name/cm/cmctl/package.nix

```diff
@@ -35,36 +31,25 @@ buildGoModule rec {
     installShellFiles
   ];
 
-  # Trusted by this computer: no: x509: “cert-manager” certificate is not trusted
-  doCheck = !stdenv.hostPlatform.isDarwin;
+  checkPhase = ''
+    go test --race $(go list ./... | grep -v /test/)
+  '';
 
-  postInstall =
-    ''
-      mv $out/bin/ctl $out/bin/cmctl
-    ''
-    + lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
-      installShellCompletion --cmd cmctl \
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    installShellCompletion --cmd cmctl \
         --bash <($out/bin/cmctl completion bash) \
         --fish <($out/bin/cmctl completion fish) \
         --zsh <($out/bin/cmctl completion zsh)
-    '';
-
-  passthru.updateScript = nix-update-script { };
+  '';
 
   meta = with lib; {
-    description = "CLI tool for managing cert-manager service on Kubernetes clusters";
+    description = "`cmctl` is the command line utility that makes cert-manager'ing easier.";
     mainProgram = "cmctl";
     longDescription = ''
-      cert-manager adds certificates and certificate issuers as resource types
-      in Kubernetes clusters, and simplifies the process of obtaining, renewing
-      and using those certificates.
-
-      It can issue certificates from a variety of supported sources, including
-      Let's Encrypt, HashiCorp Vault, and Venafi as well as private PKI, and it
-      ensures certificates remain valid and up to date, attempting to renew
-      certificates at an appropriate time before expiry.
+      cmctl is a command line tool that can help you manage cert-manager and
```

Comment:
Is the previous longDescription incorrect?

Reply:
Updated to follow the new GitHub README description since the CLI tool has moved out of the main repo page.

Reply:
Yes, sure, but why would you remove the previous description? It seems useful to me?

Reply:
Yes, that's fair. I'll add it back!